save-demo-agent/build.gradle.kts

```diff
@@ -58,4 +59,18 @@ kotlin {
             }
         }
     }
+
+    @Suppress("GENERIC_VARIABLE_WRONG_DECLARATION")
```

Comment:
still need this suppress? you already provide a type `TaskProvider<KotlinNativeLink>`?

Reply:
As I understand, due to kts features, we still need to pass a type to `tasks.named`, but diktat fixes it into 
```kotlin
val linkTask: TaskProvider<KotlinNativeLink> = tasks.named("linkReleaseExecutableLinuxX64")
```
which is wrong.

Reply:
So yes, we still need it